pom.xml

```diff
@@ -132,7 +132,7 @@
         <dependency>
             <groupId>io.appium</groupId>
             <artifactId>java-client</artifactId>
-            <version>7.5.0</version>
+            <version>8.0.0</version>
```

Comment:
`8.2.0` is available

Reply:
Unless there are really important changes, Id rather keep it 1 or 2 subversions lower. Is there any mayor reason?

Reply:
Is there any reason to not have the latest available version? 😄

Reply:
unstabilities :D